tools/topology/topology2/cavs-windows-simple-pipeline.conf

```diff
@@ -94,6 +95,7 @@ Object.Pipeline {
                 }
 
 		format s16le
+		index 2
```

Comment:
do we need to add index for the following pipeline (mixout-gain-playback 3) ?

Reply:
yes, fixed now. thanks!